credentials/alts/internal/handshaker/handshaker.go

```diff
@@ -41,9 +43,10 @@ const (
 	// The maximum byte size of receive frames.
 	frameLimit              = 64 * 1024 // 64 KB
 	rekeyRecordProtocolName = "ALTSRP_GCM_AES128_REKEY"
-	// maxPendingHandshakes represents the maximum number of concurrent
+	// defaultMaxPendingHandshakes represents the default maximum number of concurrent
 	// handshakes.
-	maxPendingHandshakes = 100
+	defaultMaxPendingHandshakes        = 100
+	maxConcurrentHandshakesEnvVariable = "GRPC_ALTS_MAX_CONCURRENT_HANDSHAKES"
```

Comment:
Please take a look at the following files to get an idea of how we handle env vars in grpc-go:
https://github.com/grpc/grpc-go/blob/master/internal/envconfig/envconfig.go
https://github.com/grpc/grpc-go/blob/master/internal/envconfig/xds.go

Reply:
Thanks for the pointers! I've cleaned up the PR accordingly. :)